server/verify-brackets.js

```diff
@@ -0,0 +1,18 @@
+const config = require('server/routes/config');
```

Comment:
The way that node does its module loading is that you can pass it a module name, like `is-reachable` below, or you can pass it a path, which needs to start with `.` or `..` or the like.  You probably want this:

```js
const config = require('./routes/config');
```

Reply:
Oh ok I was using the wrong path. I looked up the error and saw people using "./" but I was doing "./server/routes/config".